include/dos_inc.h

```diff
@@ -123,6 +123,9 @@ bool DOS_ForceDuplicateEntry(uint16_t entry,uint16_t newentry);
 bool DOS_GetFileDate(uint16_t entry, uint16_t* otime, uint16_t* odate);
 bool DOS_SetFileDate(uint16_t entry, uint16_t ntime, uint16_t ndate);
 
+uint16_t DOS_GetSystemTimePacked();
+uint16_t DOS_GetSystemDatePacked();
```

Comment:
DOS PC's have the RTC or system clock (backed by a real crystal and battery), and the BIOS clock.

When the PC starts, the BIOS reads the ticks from the system's clock to sync-up, and then BIOS clock is set off on its own course, ticking ~18.2 times every second.

Over time, errors that play into the 18.2 Hz will cause the the BIOS clock drifts from the very accurate quartz-based RTC.

All that to say, these functions are getting the BIOS time, not the system (or RTC) time. 

So suggest they be renamed to BIOS:

This is quick with the `git replace` bash function:

```
grp DOS_GetSystemTimePacked DOS_GetBiosTimePacked
grp DOS_GetSystemDatePacked DOS_GetBiosDatePacked
```

(Here it is :-))

``` shell
# git replace
grp() {
    local -r search="$1"
    local -r replace="$2"
    shift; shift

    sed -i "s^${search}^${replace}^g" $(git grep -l "${search}" "$@")
}
```

Reply:
The 18.2 Hz rate comes from the motherboard primary crystal running at 315 MHz being divided way down by a bunch of oscillators: 
1. The NTSC Sub Carrier /22 divider, and
2. The NTSC Color Burst /4 divider, and
3. The PIT /3 divider, and
4. The clock tick /65536 divider

315000000 / (22 * 4 * 3 * 65536) = 18.20650967684659

That's all driven by hardware oscillators generating the timer interrupt (IRQ1) at that exact rate :sweat_smile: 

(those above ratios can be used as a std::chrono ratio to compute times, which I have in a branch `kc/time-sync-3`.. unfortunately no time to progress it)